.github/CODEOWNERS

```
@@ -19,4 +19,4 @@
 /src/Servers/                   @tratcher @jkotalik @anurse @halter73
 /src/Middleware/Rewrite         @jkotalik @anurse
 /src/Middleware/HttpsPolicy     @jkotalik @anurse
-/src/SignalR/                   @BrennanConroy @halter73 @anurse
+/src/SignalR/                   @mikaelm12 @BrennanConroy @halter73 @anurse
```

I'm guessing @mikaelm12 doesn't want to be re-added as a code owner.

```suggestion
/src/SignalR/                   @BrennanConroy @halter73 @anurse
```